src/shared/hooks/useNSQuery.ts

```diff
@@ -57,7 +52,7 @@ const useNSQuery = ({
       query,
       columns: withColumns ? queryColumns : undefined,
       selectedFacets,
-      facets: withFacets ? undefined : undefined,
+      facets: withFacets ? undefined : [],
```

Comment:
I was looking at that and thinking it's not obvious that passing `undefined` means it will get the default for the current namespace. As there's not many calls of `getAPIQueryUrl`, I wonder if we should pass the default at the time of the call instead? What do you think?

Reply:
Probably we should eventually consolidate the approach we take for parameters for all of these URL builder functions in general. If we change something for getAPIQueryUrl we also need to change the way getAccessionsURL handles facets, because it's not consistent either.

Reply:
Ok, so we merge?

Reply:
I guess so.
We might need a Jira for review of all of these later on, or we might need to tackle that part of the BLAST against UniParc/UniRef as it might affect getAccessionsURL (we'll use accession endpoint, but it's assuming default facets for UniProtKB at the moment)

Reply:
Good idea, I'll add a subtask to the issue.